apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.cs

```diff
@@ -346,10 +378,12 @@ public override object ExecuteScalar() => ExecuteScalarAsync(_synchronousCancell
             .ResultWithUnwrappedExceptions();
 
         /// <summary>
+        /// Executes the query and returns the first column of the first row in the result set returned by the query.
+        /// All other columns and rows are ignored. The return value is converted to type T if possible.
         /// </summary>
-        /// <param name="cancellationToken"></param>
-        /// <typeparam name="T"></typeparam>
-        /// <returns></returns>
+        /// <param name="cancellationToken">An optional token for canceling the call.</param>
+        /// <typeparam name="T">The expected return Type. If possible the return type will be converted to this Type.</typeparam>
```

Comment:
Nit: I'd say "type" rather than Type, as it's not like we're passing in a System.Type.
Nit: "If possible the return type" -> "If possible the value"
Also, what happens if it's *not* possible to convert the value? Presumably an exception? Let's document that.